config/runtime.exs

```diff
@@ -344,11 +344,14 @@ config :lightning, Lightning.Repo,
   queue_target: env!("DATABASE_QUEUE_TARGET", :integer, 50),
   queue_interval: env!("DATABASE_QUEUE_INTERVAL", :integer, 1000)
 
-url_scheme = env!("URL_SCHEME", :string, "https")
 host = env!("URL_HOST", :string, "example.com")
 port = env!("PORT", :integer, 4000)
 url_port = env!("URL_PORT", :integer, 443)
 
+config :lightning,
+  cors_origin:
+    env!("CORS_ORIGIN", :string, "*") |> String.split(",") |> List.wrap()
```

Comment:
Should we maybe we should default to the `endpoint url` instead?

Reply:
Oh sorry, to add on why `cors_plug` vs `corsica`, we wanted to be able to configure this from `runtime.exs` which I couldn't find a way to do that easily with corsica, I now see in the docs you can give the `origin` key an MFA [here](https://hexdocs.pm/corsica/Corsica.html#t:origin/0) so we could use it... 

If `cors_plug` gives us even a hint of an issue, happy to try our corsica.

Reply:
Aaah, makes sense